keel-echo/src/main/kotlin/com/netflix/spinnaker/keel/echo/UnhealthyNotificationListener.kt

```diff
@@ -48,7 +48,7 @@ class UnhealthyNotificationListener(
         val unhealthyDuration = unhealthyRepository.durationUnhealthy(event.resource)
         if (unhealthyDuration > config.minUnhealthyDuration) {
           publisher.publishEvent(
-            NotificationEvent(RESOURCE, event.resource.id, UNHEALTHY_RESOURCE, message(event, unhealthyDuration))
+            UnhealthyNotification(event.resource.id,  message(event, unhealthyDuration))
```

Comment:
I think this needs to be `RepeatedNotifaction...` or whatever the class is called, because `Notifier` is listening for the generic version of the event

Reply:
I think the listener will to all `RepeatedNotifaction` events, will double-check.